tests/components/igloohome/test_config_flow.py

```diff
@@ -0,0 +1,78 @@
+"""Test the igloohome config flow."""
+
+from unittest.mock import AsyncMock
+
+from aiohttp import ClientError
+from igloohome_api import AuthException
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.igloohome.const import DOMAIN
+from homeassistant.const import CONF_CLIENT_ID, CONF_CLIENT_SECRET
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+FORM_USER_INPUT = {
+    CONF_CLIENT_ID: "test-client-id",
+    CONF_CLIENT_SECRET: "test-client-secret",
+}
+
+
+async def test_form(
+    hass: HomeAssistant, mock_setup_entry: AsyncMock, mock_auth
+) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
+    assert result["errors"] == {}
+
+    with mock_auth:
```

Comment:
since the fixture appears to be working in the test below you don't need that here, params is enough